data/config-schema/src/main/resources/net/corda/schema/configuration/db/1.0/corda.db.json

```diff
@@ -51,6 +51,19 @@
               "type": "integer",
               "minimum": 1,
               "default": 10
+            },
+            "min_size": {
+              "description": "The minimum database pool size.",
```

Comment:
Maybe worth mentioning here that if left null then means minimumPoolSize = maximumPoolSize.